modules/updating-eus-to-eus-layered-products.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * updating/preparing-eus-eus-upgrade.adoc
+
+:_content-type: PROCEDURE
+[id="updating-eus-to-eus-olm-operators_{context}"]
+= EUS-to-EUS update for layered products and Operators installed through Operator Lifecycle Manager (OLM) 
+
+In addition to the EUS-to-EUS update steps mentioned for the web console and CLI, there are a couple of additional steps to consider when performing EUS-to-EUS updates for the clusters of layered products and those of Operators that have been installed through the Operator Lifecycle Manager (OLM). 
```

Comment:
This is kind of a long sentence (though not that bad). Might not be an improvement, but a fly-by idea to shorten/break-up:
```suggestion
In addition to the EUS-to-EUS update steps mentioned for the web console and CLI, there are additional steps to consider when performing EUS-to-EUS updates for clusters with the following:
* Layered products 
* Operators installed through Operator Lifecycle Manager (OLM)
```

Reply:
nice :) thanks!